core/src/main/java/org/eclipse/hono/util/NotificationDeviceCommandReadyObject.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+
+package org.eclipse.hono.util;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+/**
+ * Encapsulates the device command readiness notification that is defined in Hono's
+ * <a href="https://www.eclipse.org/hono/api/event-api/">Event API</a>.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public final class NotificationDeviceCommandReadyObject {
```

Comment:
this looks more like the representation of a generic *Notification*.

The only data relevant for indicating that a device is ready to receive a command seems to be *timeToBeReady* (if I understand its purpose correctly in that it indicates the number of seconds the device will be *online*).